### ISSUE TITLE
Re-design UI for Help Centre and in context help buttons

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#207]
Hi @kevincrepin, just met with @anettlinno and we'd like to request designs for: 

- **The Help Centre UI** (both the side panel and button) within the platform. I.e. how should the existing design be adapted to blend in with the rest.
- **The in-context help**. How will this look/work. What will the little buttons look like and what will happen when you click on them. (Will it open a mini pop-up when you click/hover, or will it automatically open the Help Panel on a page with the guidance).

Thank you :)

**Comment by kevincrepin** [comment]
No reason to make any changes to the logic or layout. I would only change the colour scheme a bit to fit in the new platform.

Bottom right HC button blue change: https://www.figma.com/file/gOJ4i00538e0qaP6IKIPVE/Platform-2.0-UI?type=design&node-id=7017%3A67986&mode=design&t=qWRYXOj6fiKZwaEM-1

For the actual HC, mainly just shifting awya from the bright blues and adapting the darker blue we have in the platform: https://www.figma.com/file/gOJ4i00538e0qaP6IKIPVE/Platform-2.0-UI?type=design&node-id=7017%3A68925&mode=design&t=qWRYXOj6fiKZwaEM-1

For contextual help I would have tooltips, ie: https://www.figma.com/file/gOJ4i00538e0qaP6IKIPVE/Platform-2.0-UI?type=design&node-id=7017%3A65745&mode=design&t=qWRYXOj6fiKZwaEM-1

And if the topic is too complex to explain within the tooltip, we can give an intro to this topic and have a link to the HC. For simpler explanations we can keep the user in their flow and just explain everything in the tooltip. This delegation example I worked out is a good reference for smt that can be explained a bit, but if you want more in-depth then you can go to HC.

So basically I would never lead a person straight to the HC from a contextual help / tooltip, because it's a bit unexpected behaviour. Rather open up the tooltip, epxlain a bit (or fully if short) and then offer more help with a link to the HC.

**Comment by BeccaMelhuish** [comment]
Wonderful, thank you @kevincrepin - this all looks/sounds good to me! 

@ilmartyrk, @devosama1, @ssin1901 - tagging you just for your info :)

**Comment by anettlinno** [comment]
Triage 95. @BeccaMelhuish has created list of features where to use in context help buttons but this should be revised. Will do it in Phase II and sending the issue there.
